README.md

```diff
@@ -145,6 +145,22 @@ You are ready to run cypress commands like:
 npm run cy:open
 ```
 
+### Usage with docker
+
+Build docker image:
+
+```shell
+docker build --tag saleor-dashboard .
+```
+
+Run nginx from docker and bind it to port on your machine (in this example 8080):
+
+```shell
+docker run --publish 8080:80 --env "API_URL=<YOUR_API_URL>" saleor-dashboard
+```
+
+Enter `http://localhost:8080/` to use dashboard.
+
```

Comment:
would be nice to attach also a command for replacing API URL within runtime, I think it will be something like:

```
docker exec -it -e API_URL=some url [container id] /replace-api-url.sh
```

nginx/browser will cache index.html so I think the user will see that the codebase has changed (notification on the dashboard side), but after refreshing (clicking reload on the notification) the API will be changed - would be nice also to improve that afterward (change API applies instantly)